### PR TITLE
update logging user and logrotate settings in line with upstream

### PR DESCRIPTION
**Pull request by 0atman**
Updated log settings to be similar to https://github.com/absoludity/charm-ansible-roles/commit/509a642c4850bfc1d1cfab63769bafb8e5ec5d6d

### CODE REVIEW

**Comment by 0atman** [comment]
rsync.d fixes have to be in the spec that uses this charm, I think

File: roles/wsgi-app/tasks/setup-machine.yml [hunk 1]
```diff
@@ -39,6 +39,6 @@
   template:
     src: "log-rotate.j2"
     dest: /etc/logrotate.d/{{ app_label }}
-    owner: "{{ wsgi_user }}"
-    group: "{{ wsgi_group }}"
+    owner: "{{ root }}"
```
Comment (nottrobin) [inline roles/wsgi-app/tasks/setup-machine.yml]: I suspect you haven't tested this. As far as I'm aware, `root` doesn't exist as a variable.

**Comment by nottrobin** [comment]
Made one comment. I can help you actually test this out locally if you like. Shouldn't be too difficult, especially as you're actually using the machine that's intended for running juju.

**Comment by 0atman** [comment]
Good spot! how silly. Fixed and tested:
```
# ls -al 
total 52
drwxr-xr-x  2 root root 4096 May 20 15:15 .
drwxr-xr-x 87 root root 4096 May 20 15:14 ..
-rw-r--r--  1 root root  126 Oct 30  2014 apport
-rw-r--r--  1 root root  173 Apr 20  2012 apt
-rw-r--r--  1 root root   79 Apr 19  2013 aptitude
-rw-r--r--  1 root root  232 Apr 13  2012 dpkg
-rw-r--r--  1 root root  609 Jan 21 17:30 landscape-client
-rw-r--r--  1 root root   94 Apr 21 18:33 ppp
-rw-r--r--  1 root root  515 Mar 30  2012 rsyslog
-rw-r--r--  1 root root  178 Apr  5  2012 ufw
-rw-r--r--  1 root root  114 Oct  7  2011 unattended-upgrades
-rw-r--r--  1 root root  122 Jan 18  2013 upstart
-rw-r--r--  1 root root  280 May 20 15:13 wsgi-app-0
```

**Comment by 0atman** [comment]
@nottrobin same with this one, is this needed? It might be a dupe

**Comment by nottrobin** [comment]
@0atman This one is definitely already implemented. Go ahead and delete.

**Pull request closed without merging**